.gitmodules

```diff
@@ -1,9 +1,9 @@
 [submodule "mshadow"]
 	path = mshadow
-	url = https://github.com/larroy/mshadow.git
+	url = https://github.com/dmlc/mshadow.git
```

Comment:
Is this change sufficient to change the submodule? Do you have to do `git add mshadow`/ `git add dmlc-core` to specify a particular commit id for each one of them?

Reply:
Yes they have to be updated. See the related PRS linked in the description please.